### ISSUE TITLE
Icon Request: MyPaint

### ISSUE THREAD

**Issue by w9** [snwh/paper-icon-theme#140]
Application name: MyPaint
File name: `mypaint.png`
Description: Currently it is linked to a generic paint palette icon, I think this is an amazing piece of software and it deserves a more specific icon. :)

Reference:

![image](https://cloud.githubusercontent.com/assets/4512709/14996858/c892434c-1117-11e6-9236-b8c7be96f51a.png)

**Comment by snwh** [comment]
I agree, but as of right now the goal is to expand Paper's coverage.
